README.md

```diff
@@ -77,17 +77,44 @@ npm run deploy
 
 You will need to have write privileges to this repository on Github to be able to do this.
 
-**NOTE: Use caution when doing this**, *before deploying you should make sure your build is successful and runs as expected. You may do this by running the script `start:prod` which will create a production build then start a local server in the dist directory.*
+**NOTE: Use caution when doing this**, _before deploying you should make sure your build is successful and runs as expected. You may do this by running the script `start:prod` which will create a production build then start a local server in the dist directory._
 
 Assuming you have already run the `build` script, you may view the site using the output / bundled files from Webpack by doing:
 
 ```
 yarn start:prod
 ```
+
 or
 
 ```
 npm run start:prod
 ```
 
 You may now view the site on `http://localhost:5000`
+
```

Comment:
Nice.  Thank you for the in-depth instructions.

Reply:
The only thing in the README that I can think of that would also be helpful would be to have an explanation for how to test new languages.

Reply:
Good point, will add.